package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@nrwl/nx-source",
-  "version": "10.2.1",
+  "version": "999.9.8",
```

Comment:
This should not be checked in

Reply:
You are right. I didn't notice this. I'll remove it thanks!

I'm going to take a second look at this anyway and possibly make some changes to it.